kazoo/client.py

```diff
@@ -120,6 +120,7 @@ def __init__(
         ca=None,
         use_ssl=False,
         verify_certs=True,
+        check_hostname=False,
```

Comment:
Can you please add a description of this new param L185? This is useful for the documentation.

Reply:
I have added a line to the description.